components/shapediver/parameter/select/SelectFullWidthCards.tsx

```diff
@@ -0,0 +1,78 @@
+import {Card, Group, Image, Stack, Text} from "@mantine/core";
+import React from "react";
+import {SelectComponentProps} from "./SelectComponent";
+import classes from "./SelectFullWidthCards.module.css";
+
+/**
+ * Functional component that displays selectable full-width cards with images and descriptions.
+ * Each card is displayed on a separate line, and the selected card is highlighted with a thicker border.
+ */
+export default function SelectFullWidthCards(props: SelectComponentProps) {
+	const {value, onChange, items, disabled, itemData} = props;
+
+	// Transform items array into the format expected by the component
+	const cardData = items.map((item) => {
+		const data = itemData?.[item];
+		return {
+			value: item,
+			label: data?.displayname || item,
+			description: data?.description,
+			imageUrl: data?.imageUrl,
+			width: data?.width || "100px",
+			color: data?.color,
+		};
+	});
+
+	// Handle card selection
+	const handleCardClick = (cardValue: string) => {
+		if (!disabled) {
+			onChange(cardValue);
+		}
+	};
+
+	return (
+		<Stack>
+			{cardData.map((card) => (
+				<Card
+					key={card.value}
+					shadow="none"
+					radius="md"
+					className={`${classes.card} ${disabled ? classes.cardDisabled : ""}`}
+					withBorder
```

Comment:
@LemToUp let's avoid style properties like shadow, radius, and withBorder unless we don't want them to be changeable by the user. Whenever possible let's not define them, such that this can be changed using the theme.

Reply:
Refactored. Should be better now.